autoload/codefmt/zprint.vim

```diff
@@ -0,0 +1,82 @@
+" Copyright 2019 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+let s:supported_filetypes = ['clojure']
```

Comment:
Style nit: Is this meant to be constant or updateable? If constant, consider capitalizing and doing `lockvar!` since vimscript functions make it easy to accidentally modify lists otherwise (and you'll probably want an `if exists()` guard since this is an autoload file). Otherwise, you could attach it to `s:plugin.globals`.

Reply:
That's from `autoload/codefmt/prettier.vim` but it would make more sense to use the simpler `AppliesToBuffer` function from `autoload/codefmt/rustfmt.vim` since there's just one filetype.

Reply:
Done